lib/taskana-core/src/main/java/pro/taskana/common/internal/util/LogSanitizer.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.common.internal.util;
+
+public class LogSanitizer {
```

Comment:
Is this used in taskana-core? 
@gitgoodjhe should we move this class to taskana-rest-spring?

Reply:
Good question, not yet used in core, but might be in the future?

Reply:
> 
User provided data, such as URL parameters, POST data payloads or cookies, should always be considered untrusted and tainted. Applications logging tainted data could enable an attacker to inject characters that would break the log file pattern. This could be used to block monitors and SIEM (Security Information and Event Management) systems from detecting other malicious events.

This problem could be mitigated by sanitizing the user provided data before logging it.

A dedicated rest issue. Will never happen in core. 
I am for moving this util class to taskana-rest-spring while keeping the same package structure

Reply:
alright